cwf/gateway/fabfile.py

```diff
@@ -38,6 +38,7 @@ class SubTests(Enum):
     GX = "gx"
     GY = "gy"
     QOS = "qos"
+    OMNI = "omnipresent"
```

Comment:
Let's get rid of this too. We have multiple ways of running specific tests. One is the `tests_to_run` tag and the other is `test_re`. The `test_re` tag uses regex to match tests names. So in this case, `fab integ_test:test_re=TestOmni`, should do the trick.